packages/@rnw-scripts/just-task/just-task.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ * @ts-check
+ */
+
+const {cleanTask, eslintTask, jestTask, series, task, tscTask} = require('just-scripts');
+
+task('clean', cleanTask(['lib-commonjs']));
+
+task('ts:commonjs', tscTask({ module: 'commonjs', outDir: 'lib-commonjs' }));
+task('ts', series('ts:commonjs'));
+
+task('jest', jestTask());
+task('jest:watch', jestTask({ watch: true }));
+
+task('build', series('ts'));
```

Comment:
This is no longer the lib preset bc I wanted to avoid the overhead of building CommonJS + ESModule when we only use one and are adding more TS.